tests/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:lts-alpine3.14
```

Comment:
Good point. Done.

While the test container is a separate thing, it makes sense to be consistent, plus it avoids all the package-lock.json is out of date errors.

I would like to update all these packages and take advantage of some changes in Cucumber and Puppeteer to make life easier, but there have been some major changes and it would take a bit of unpicking.